src/main/java/org/kainos/ea/resources/EmployeeController.java

```diff
@@ -29,6 +31,23 @@ public Response getAllEmployees () {
             return Response.serverError().build();
         }
     }
+
+    @PUT
+    @Path("/employee/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response updateEmployee(@PathParam("id") int id, EmployeeRequest employee){
```

Comment:
I assume EmployeeRequest has national insurance number? you will need a new model if so